docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,689 @@
+
+==== Building and Deploying the Helidon Teller Application
+
+Download the image for MicroTx Free from https://container-registry.oracle.com/[Oracle Container Registry] and add the `TmmLib` library to the local Maven repository. This library is available in the `<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/` location.
+
+Install the OTMM library to the local Maven repository using the following command:
+
+[source,shell]
+----
+mvn install:install-file -Dfile=<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/TmmLib-22.3.2.jar -DpomFile=<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/TmmLib-weblogic-22.3.2.pom
+----
+
+Create the Docker image for the Helidon Teller Application and deploy it in the Kubernetes cluster.
+
+===== Creating the Docker Image
+
+In the MicroTx binaries, the Helidon Teller Application samples are available in the `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/teller` location.
+
+Update the `application.yaml` file available in the <OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/teller/src/main/resources location. In this file, specify the endpoint URLs of the Helidon Teller Application along with the authentication details, if any.
+
+[source,yaml]
+----
+helidonServiceEndpoint: http://<HELIDON_APP_K8S_SERVICE_NAME>:<HELIDON_APP_K8S_SERVICE_PORT>/<HELIDON_APP_CONTEXT_PATH>
+weblogicServiceEndpoint: http://<WEBLOGIC_APP_K8S_SERVICE_NAME>:<WEBLOGIC_APP_K8S_SERVICE_PORT>/<WEBLOGIC_APP_CONTEXT_PATH>
+Weblogic:
+  BasicAuth:
+    UserName: <wls_username>
+    Password: <wls_password>
+----
+
+For example:
+
+[source,yaml]
+----
+helidonServiceEndpoint: http://microtx-helidon.helidon-ns.svc.cluster.local:8084/mtxhelidon
+weblogicServiceEndpoint: http://wls-domain-cluster-cluster-1.wls-domain-ns.svc.cluster.local:8001/mtxwls
+Weblogic:
+  BasicAuth:
+    UserName: <wls_username>
+    Password: <wls_password>
+----
+Update the `tmm.properties` file available in the `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/teller/src/main/resources` location.
+
+[source,java]
+
+----
+### oracle.tmm.TcsUrl value refers to the MicroTx coordinator URL
+oracle.tmm.TcsUrl = http(s)://<OTMM_K8S_SERVICE_NAME>:<OTMM_SERVICE_PORT>/api/v1
+oracle.tmm.TcsConnPoolSize = 15
+### oracle.tmm.CallbackUrl refers to the Helidon Teller Application call-back URL along with the context path
+oracle.tmm.CallbackUrl = http://<HELIDON_TELLER_APP_K8s_SERVICE_NAME>:<HELIDON_TELLER_APP_K8s_SERVICE_NAME>/<HELIDON_TELLER_APP_CONTEXTPATH>
+ 
+## Transaction Timeout Out value in milli seconds
+oracle.tmm.TransactionTimeout = 60000
+ 
+oracle.tmm.PropagateTraceHeaders = false
+----
+
+Create the Docker image for the Helidon Teller Application, upload the image to the Docker registry, and then deploy the application in the Kubernetes cluster.
```

Comment:
How do they create the Docker image? Shouldn't they run the docker build command for this on a DockerFile? I looked at the readme.md in `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/teller` and it has instruction on how to run the docker build to create the image. The directory also has the Dockerfile that would allow to build the image, so should it not be given as an example instruction? Like:

1. Go to the teller directory:
    ```
    cd <OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/teller
    ```
2. Run the docker build command to create the image (fill the image_name and tag with the desired values):
    ```
   docker build --network host -t <image_name>:<tag> .
   ```